CHANGELOG.md

```diff
@@ -13,10 +13,16 @@
 - oneview_server_certificate
 - oneview_server_hardware
 - oneview_server_hardware_type
+- oneview_server_profile
+- oneview_server_profile_template
 - oneview_storage_pool
 - oneview_storage_system
+- oneview_storage_volume
 - oneview_volume_template
 
+Adds API1600 support to HPE Synergy Image Streamer resources:
+  - image_streamer_deployment_plan
```

Comment:
if we are listing image streamer resource separately, can you add the artifact bundles resource also in the same way.